k8s/openebs-operator.yaml

```diff
@@ -152,6 +158,29 @@ spec:
         - name: MAYA_PORTAL_URL
           value: "https://mayaonline.io/"
 ---
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: snapshot-controller
+  namespace: default
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: snapshot-controller
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+        - name: snapshot-controller
+          image: openebs/k8s-snapshot-controller:ci
```

Comment:
Using "ci" tag is not recommended. Probably this PR needs to wait till an initial version of the snapshot provisioner is checked in and a named tag is available.

Reply:
Yes, once we have a release we can update the PR with stable release tag.